homeassistant/components/tplink/strings.json

```diff
@@ -68,6 +68,33 @@
       },
       "overheated": {
         "name": "Overheated"
+      },
+      "battery_low": {
+        "name": "Battery low"
+      },
+      "cloud_connection": {
+        "name": "Cloud connection"
+      },
+      "update_available": {
+        "name": "[%key:component::binary_sensor::entity_component::update::name%]",
+        "state": {
+          "off": "[%key:component::binary_sensor::entity_component::update::state::off%]",
+          "on": "[%key:component::binary_sensor::entity_component::update::state::on%]"
+        }
+      },
+      "is_open": {
+        "name": "[%key:component::binary_sensor::entity_component::door::name%]",
+        "state": {
+          "off": "[%key:common::state::closed%]",
+          "on": "[%key:common::state::open%]"
+        }
+      },
```

Comment:
It would be nicer if this was automatic based on the device class, but I do see this pattern from #89702 so I think its what we have for now